lib/gulp-html5-lint.js

```diff
@@ -5,14 +5,35 @@ var _ = require('ls-lodash'),
     htmlLint = require('html5-lint'),
     gutil = require('gulp-util'),
     PluginError = gutil.PluginError,
-    pluginName = require('../package').name;
+    pluginName = require('../package').name,
+    request = require('sync-request');
 
 module.exports = function gulpHtml5Lint(options) {
     var linterOpts = _.safeMerge(options),
-        errorOutputList = [];
+        errorOutputList = [],
+        req,
+        apiAvailable;
+
+    try {
+        req = request('GET', linterOpts.website || 'https://html5.validator.nu', {
+            timeout: 3000
+        });
+    }
+    catch (e) {
+        req = {
+            statusCode: 0
+        };
+    }
+    finally {
+        apiAvailable = req.statusCode === 200;
+    }
+
+    if (!apiAvailable) {
+        gutil.log('html-lint API is DOWN. Skipping linting.');
```

Comment:
I think we need an option that flags whether API down is an error or just a warning (as it is here).  For our workflow, I would want the CI pipeline to stop if all linters were not run, but it also makes total sense that there are many workflows where you don't want to fail when the linter cannot run.